docs/crocks/Pair.md

```diff
@@ -0,0 +1,85 @@
+# Pair
+
+`Pair c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pair` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of), [`type`](#type) | [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`concat`](#concat), [`equals`](#equals), [`fst`](#fst), [`inspect`](#inspect),  [`map`](#map), [`merge`](#merge), [`of`](#of), [`type`](#type), [`snd`](#snd), [`swap`](#swap), [`value`](#value) |
+
+## Constructor
+
+### of
+
+`Pair m => c a -> m c a`
+
+### type
+
+`() -> String`
+
+## Instance
+
+### ap
+
+`Pair m => m c (a -> b) ~> m c a -> m c b`
+
+### bimap
+
+`Pair m => m c a ~> (c -> d) -> (a -> b) -> m d b`
+
+### chain
+
+`Pair m => m c a ~> (a -> m c b) -> m c b`
+
+### concat
+
+`Pair m => m c a ~> m c a -> m c a`
+
+### equals
+
+`a -> Boolean`
+
+### fst
+
+`Pair m => m c a ~> m c a -> c`
+
+### inspect
+
+`() => String`
+
+### map
+
+`Pair m => m c a ~> (a -> b) -> m a b`
+
+### merge
+
+`Pair m => (a -> b -> c) -> m a b -> c`
+
+### of
+
+`Pair m => a -> m () a`
```

Comment:
Looks like it was closer here then on the Constructor version. But this is not needed.